nats/src/connector.rs

```diff
@@ -603,9 +604,18 @@ impl ServerAddress {
         self.0.scheme() == "tls"
     }
 
-    /// Returns if the server url had embedded username and password.
-    pub fn has_user_pass(&self) -> bool {
```

Comment:
Public method, so probably don't want to be removing that outright.

Reply:
Agreed, I didn't mean to remove that. I've added it back.